admission.html

```diff
@@ -0,0 +1,139 @@
+                    </div>
+                    <fieldset class="group">
+                        <legend class="group-label">Both courses and programs</legend>
+                        <div class="group-option-radio">
+                            <input type="radio" id="typeboth" name="type" value="both" checked class="input-radio" checked>
+                            <label for="typeboth" class="radio-label">Both</label>
+                            <input type="radio" id="typecourses" name="type" value="course" class="input-radio"> 
+                            <label for="typecourses" class="radio-label">Courses</label>
+                            <input type="radio" id="typeprograms" name="type" value="program" class="input-radio">
+                            <label for="typeprograms" class="radio-label">Programs</label>
+                        </div>
+                    </fieldset>
+                     <div class="horizontalLine"></div>
+                     <fieldset class="group">
+                        <legend class="group-label">Course level</legend>
+                        <div class="checkbox-row">
+                            <label class="checkbox-container">Beginner level
+                                <input type="checkbox" id="basicLevel" name="level" value="beginner" class="level-checkbox"> 
+                                <span class="checkmark"></span>
+                            </label>
+                            <i class="fas fa-question-circle tooltip">
+                                <span class="tooltiptext">You do not need previous higher education studies but must have successfully completed upper secondary studies.</span>
+                            </i>
+                            <div class="tooltip"></div>
+                        </div>
+                        <div class="checkbox-row">
+                            <label class="checkbox-container">Bachelor's level
+                                <input type="checkbox" id="mediumLevel" name="level" value="bachelor" class="level-checkbox"> 
+                                <span class="checkmark"></span>
+                            </label>
+                            <i class="fas fa-question-circle tooltip">
+                                <span class="tooltiptext">You must have successfully completed upper secondary studies.</span>
+                            </i>
+                            <div class="tooltip"></div>
+                        </div>
+                        <div class="checkbox-row">
+                            <label class="checkbox-container">Master's level
+                                <input type="checkbox" id="advancedLevel" name="level" value="master" class="level-checkbox"> 
+                                <span class="checkmark"></span>
+                            </label>
+                            <i class="fas fa-question-circle tooltip">
```

Comment:
```suggestion
                            <i class="fa fa-question-circle tooltip">
```


though I don't know, maybe fas is intended. it's just usually it's fa...